packages/terra-application-navigation/src/drawer-menu/_DrawerMenuFooterButton.jsx

```diff
@@ -19,14 +19,15 @@ const propTypes = {
   onClick: PropTypes.func,
 };
 
-const DrawerMenuFooterButton = ({ text, onClick }) => (
+const DrawerMenuFooterButton = ({ text, onClick, ...customProps }) => (
   <button
     className={cx('drawer-menu-footer-button')}
     type="button"
     onClick={onClick}
     onBlur={enableFocusStyles}
     onMouseDown={disableFocusStyles}
     data-focus-styles-enabled
+    {...customProps}
```

Comment:
Generally custom props should always be spread first so they don't overwrite the props the component relies upon.  Here for example, although unlikely in this case,  the custom props could theoretically overwrite the click, blur, and mouse down callbacks.

Reply:
That makes sense, Updated here aa934352b61a81e007fbb26d35fefff310f16d02